ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -408,6 +408,13 @@ export default defineComponent({
           setUnsavedChanges(false)
           // Delete the draft on successful submission
           if (getMhrInformation.value.draftNumber) await deleteMhrDraft(getMhrInformation.value.draftNumber)
+          const newItem: RegTableNewItemI = {
+            addedReg: String(getMhrInformation.value.mhrNumber),
+            addedRegParent: '',
+            addedRegSummary: null,
+            prevDraft: String(getMhrInformation.value.documentId) || ''
```

Comment:
Same as above.

Reply:
done..

Reply:
Same as above

Reply:
Made change to no conversion..  Both values are strings... Not sure why I did that in the first place.